benchmarks/src/jmh/java/reactor/AssemblyTraceBenchmark.java

```diff
@@ -44,25 +45,33 @@
 
 	static final String JAVA_8 = "/Library/Java/JavaVirtualMachines/jdk1.8.0_192.jdk/Contents/Home/bin/java";
```

Comment:
why leave the two constants if the bench is otherwise edited to remove their usage?

Reply:
I agree we should actually remove this given how dependent it is on the JDK path. It should be an external tester choice to switch jvm anyway